packages/frontend/src/components/SimpleChart/index.tsx

```diff
@@ -105,42 +121,50 @@ const SimpleChart: FC<SimpleChartProps> = memo((props) => {
         [onSeriesContextMenu, eChartsOptions],
     );
 
+    const onLegendChange = useCallback((params: LegendClickEvent) => {
+        setSelectedLegends(params.selected);
+    }, []);
+
     const onEvents = useMemo(
```

Comment:
perhaps this doesn't need to be an useMemo, you can add these properties directly on `<EChartsReact`

Reply:
Sure, looking into it!